src/crush/CrushWrapper.cc

```diff
@@ -1035,6 +1035,15 @@ int CrushWrapper::detach_bucket(CephContext *cct, int item)
   return bucket_weight;
 }
 
+bool CrushWrapper::check_bucket_relation(crush_bucket *f, crush_bucket *tar){
```

Comment:
change:
bool CrushWrapper::check_bucket_relation(crush_bucket *f, crush_bucket *tar){
to:
bool CrushWrapper::check_bucket_relation(crush_bucket *f, crush_bucket *tar)
{